inc/utils/content.php

```diff
@@ -399,11 +399,9 @@ function odm_excerpt($the_post, $num = 40, $read_more = '')
 
 		$get_the_excerpt = apply_filters('translate_text', $get_the_excerpt, odm_language_manager()->get_current_language());
 
-		$excerpt = explode(' ', strip_shortcodes($get_the_excerpt), $limit);
-
+		$excerpt = explode(' ', strip_shortcodes($get_the_excerpt), -$limit);
```

Comment:
Yes, it is intended. 
I would like to get all the strings from 0 to limited number only, so it means that all the first component except the last -limit are returned.